### DISCUSSION TITLE
default date_time for migration

### DISCUSSION THREAD

**Discussion by mohammadjavad948** [SeaQL/sea-orm#974] [Q&A] [answered]
Hi i have a table like this
![image](https://user-images.githubusercontent.com/52201992/184822277-e2648ee0-9aab-4bd9-94ee-a1693bf4fb0c.png)
how can i set `CreatedAt` default to be `now()`?

**Comment by billy1624** [comment, marked as answer]
Hey @mohammadjavad948, welcome and thanks for asking!

We're adding support for the default column expression in SeaQuery
- https://github.com/SeaQL/sea-query/issues/347

AT the meantime, you can use the [extra](https://docs.rs/sea-query/0.26.2/sea_query/table/struct.ColumnDef.html#method.extra) method as a workaround. I.e. `.extra("DEFAULT NOW()")`
